backoffice/src/main/java/com/wedriveu/backoffice/model/BackOfficeModel.java

```diff
@@ -32,4 +38,33 @@ public Future getFuture() {
     public void updateCounter(VehicleCounter vehicleCounter) {
         this.vehicleCounter = vehicleCounter;
     }
+
+    public void generateMap(List<Booking> list) {
```

Comment:
Please, i think this is not something that should be done in a model/entity, why not using an utility class for the Map?